GetJobProfiles/Models/Recipe/Parts/HowToBecomePart.cs

```diff
@@ -0,0 +1,16 @@
+﻿using GetJobProfiles.Models.Recipe.Fields;
+
+namespace GetJobProfiles.Models.Recipe.Parts
+{
+    public class HowToBecomePart
+    {
+        public TabField HowToBecome { get; set; } = new TabField();
```

Comment:
Why does this new type of field have to be newed up when the built in types don't?